projects/libredwg/llvmfuzz.options

```diff
@@ -1,2 +1,4 @@
 [libfuzzer]
 detect_leaks=0
+timeout=4000
+rss_limit_mb=24000
```

Comment:
I don't think ClusterFuzz respects an rss_limit_mb in the options file that is higher than our default (I think 2048 (MB) or something in that ballpark).
This is because the machines we run on have a fixed amount of memory so there's no point in allowing a higher limit, it will just crash the machine.